spec/index.bs

```diff
@@ -7281,6 +7281,23 @@ Issue: Because timestamp query provides high-resolution GPU timestamp, we need t
 
 # Canvas Rendering &amp; Swap Chains # {#canvas-rendering}
 
+## HTMLCanvasElement.getContext ## {#canvas-getcontext}
+
+A {{GPUCanvasContext}} object can be obtained via the `getContext` method of an {{HTMLCanvasElement}} instance by
```

Comment:
can we link to the external `HTMLCanvasElement` spec here?

Reply:
see how we refer to other external specs - https://github.com/gpuweb/gpuweb/blob/dc050f0285d40d0ecbc1db0a8602d75769e2e29e/spec/index.bs#L28-L39

Reply:
I added a link to the spec to the title and description.